src/frontend/context/EndretUtbetalingAndelContext.ts

```diff
@@ -23,16 +23,20 @@ const [EndretUtbetalingAndelProvider, useEndretUtbetalingAndel] = createUseConte
                     : feil(felt, 'Du må velge en årsak'),
         });
 
-        const periodeSkalUtbetalesTilSøkerFelt = useFelt<boolean>({
-            verdi: endretUtbetalingAndel.prosent !== undefined && endretUtbetalingAndel.prosent > 0,
+        const periodeSkalUtbetalesTilSøkerFelt = useFelt<boolean | undefined>({
+            verdi:
+                endretUtbetalingAndel.prosent === undefined ||
+                endretUtbetalingAndel.prosent === null
+                    ? undefined
+                    : endretUtbetalingAndel.prosent > 0,
         });
 
         const { skjema, kanSendeSkjema, onSubmit, nullstillSkjema } = useSkjema<
             {
                 person: string | undefined;
                 fom: FamilieIsoDate | undefined;
                 tom: FamilieIsoDate | undefined;
-                periodeSkalUtbetalesTilSøker: boolean;
+                periodeSkalUtbetalesTilSøker: boolean | undefined;
```

Comment:
Forklare hvorfor man har lagt til undefined: har lagt til undefined for å håndtere initiell tilstand hvor endretUtbetalingAndel ikke har blitt oppdatert enda